### ISSUE TITLE
No kernel identified while running tensorflow with GPU

### ISSUE THREAD

**Issue by vaibhavAIJKV** [tensorflow/tensorflow#67768]
### Issue type

Bug

### Have you reproduced the bug with TensorFlow Nightly?

No

### Source

source

### TensorFlow version

2.13.0

### Custom code

Yes

### OS platform and distribution

Windows 10

### Python version

3.8.8

### CUDA/cuDNN version

cuda_12.4.r12.4/compiler.34097967_0

### GPU model and memory

Nvidia Geforce MX450

### Current behavior?

I am using windows 10 and using tensorflow with GPU using tensorflow-directml-plugin API. While running the code I am getting the following error:

> InvalidArgumentError: No OpKernel was registered to support Op 'CudnnRNNV2' used by {{node sequential/bidirectional/forward_cu_dnnlstm/CudnnRNNV2}} with these attrs: [seed=0, dropout=0, T=DT_FLOAT, input_mode="linear_input", direction="unidirectional", rnn_mode="lstm", seed2=0, is_training=true]
> Registered devices: [CPU, GPU]
> Registered kernels:
>   <no registered kernels>
> 
> 	 [[sequential/bidirectional/forward_cu_dnnlstm/CudnnRNNV2]] [Op:__inference_train_function_7434]

It's a simple model with configuration something like:
   
    regressor2.add(Bidirectional(LSTM(units = 128,  return_sequences = True)))
    regressor2.add(Flatten())    
    regressor2.add(Dense(units = 64,activation='relu'))
    regressor2.add(Dense(units = 1,activation='linear'))
    opt = keras.optimizers.Adam(learning_rate=0.001)
    regressor2.compile(optimizer = opt,  loss='mean_squared_error',metrics=["MeanSquaredError"])

### Standalone code to reproduce the issue

```shell
regressor2.add(Bidirectional(LSTM(units = 128,  return_sequences = True)))
    regressor2.add(Flatten())    
    regressor2.add(Dense(units = 64,activation='relu'))
    regressor2.add(Dense(units = 1,activation='linear'))
    opt = keras.optimizers.Adam(learning_rate=0.001)
    regressor2.compile(optimizer = opt,  loss='mean_squared_error',metrics=["MeanSquaredError"])
```

**Comment by sushreebarsa** [comment]
@vaibhavAIJKV GPU support on native-Windows is only available for 2.10 or earlier versions, starting in TF 2.11, CUDA build is not supported for Windows. For using TensorFlow GPU on Windows, you will need to build/install TensorFlow in WSL2 or use tensorflow-cpu with TensorFlow-DirectML-Plugin.
Please refer to this [doc](https://www.tensorflow.org/install/source_windows) for further information.
Thank you!

**Comment by vaibhavAIJKV** [comment]
> @vaibhavAIJKV GPU support on native-Windows is only available for 2.10 or earlier versions, starting in TF 2.11, CUDA build is not supported for Windows. For using TensorFlow GPU on Windows, you will need to build/install TensorFlow in WSL2 or use tensorflow-cpu with TensorFlow-DirectML-Plugin. Please refer to this [doc](https://www.tensorflow.org/install/source_windows) for further information. Thank you!

Hi
As mentioned I am using TensorFlow-DirectML-Plugin, but let me go through the document to see if installing other dependencies resolve this issue

**Comment by sushreebarsa** [comment]
@vaibhavAIJKV Sure, kindly let us know?
Thank you!
